packages/kwasm/v0.2.3+1/package.yaml

```diff
@@ -0,0 +1,18 @@
+# yaml-language-server: $schema=https://glasskube.dev/schemas/v1/package-manifest.json
+
+name: "kwasm"
+shortDescription: "The KWasm Operator allows for fine-grained control over node provisioning instead of using the node installer with a DaemonSet"
+iconUrl: "https://avatars.githubusercontent.com/u/113929551"
+defaultNamespace: "kwasm"
+helm:
+  repositoryUrl: "http://kwasm.sh/kwasm-operator/"
+  chartName: "kwasm-operator"
+  chartVersion: "0.2.3"
+  values:
+    kwasmOperator: 
+      installerImage: "ghcr.io/spinkube/containerd-shim-spin/node-installer:v0.15.1"
```

Comment:
What is this for? I can't find it in either the kwasm or spin documentation

Reply:
Found it here: 
![Screenshot 2024-07-31 at 13 54 35](https://github.com/user-attachments/assets/4ab4c26e-f596-40e2-b804-14b604cd4717)

Reply:
Ohh interesting… the helm chart readme doesn't mention this … 

Not sure how I feel about having a package configured especially for another package but I guess there's no way arout it here :thinking:

Reply:
Yeah it is peculiar